nixos/modules/flyingcircus/roles/statshost.nix

```diff
@@ -89,6 +89,19 @@ in
         description = "HTTP host name for the stats frontend. Must be set.";
         example = "stats.example.com";
       };
+
+      prometeusMetricRelabel = mkOption {
```

Comment:
Ah, the typo is always there. I suggest to clean it up. :)

Reply:
lalala